src/main/scala/nl.knaw.dans.easy.download/EasyDownloadServlet.scala

```diff
@@ -101,16 +101,12 @@ class EasyDownloadServlet(app: EasyDownloadApp) extends ScalatraServlet
     }
   }
 
-  private def sendOkResponse(uuid: UUID, path: Path) = {
-    val licenseLinkText = getLicenseLinkText(uuid, path).getOrElse(None)
-    licenseLinkText.foreach(response.addHeader("Link", _))
+  private def sendOkResponse(fileItem: Try[FileItem]) = {
+    getLicenseLinkText(fileItem).foreach(response.addHeader("Link", _))
     Ok()
   }
 
-  private def getLicenseLinkText(uuid: UUID, path: Path): Try[Option[String]] = {
-    for {
-      fileItem <- app.getFileItem(uuid, path)
-      licenseLinkText <- app.configuration.licenses.getLicenseLinkText(fileItem)
-    } yield licenseLinkText
+  private def getLicenseLinkText(fileItem: Try[FileItem]): Option[String] = {
+    fileItem.map(file => Some(s"""<${file.licenseKey}>; rel="license"; title="${file.licenseTitle}"""")).getOrElse(None)
```

Comment:
Am I correct in saying that when `easy-download` calls `easy-auth-info` and the latter returns an error (for any arbitrary reason), this error is not translated into an error response in the former? In other words, it does not cause a `500 Internal Server Error`, `503 Service Unavailable` or something similar?

Reply:
`fileitem` is fetched in method `getFileItem` in `AuthorisationComponent` and it returns an error if the fileitem can't be fetched.